.github/CONTRIBUTING.md

```diff
@@ -56,20 +51,7 @@ Prerequisites:
    annoyingly slow):
 
    ```shell-session
-   $ tox -e lint
-   lint create: .tox/lint
```

Comment:
Same here plz. I specifically added the output so there's an expectation that is visible.